config/functions.php

```diff
@@ -107,8 +105,8 @@ function instalar($db_ip, $db_user, $db_pass, $db_name, $db_data, $l2jruss, $adm
 	$db = strtolower($db_data) == "l2j" ? true : false;
 	$adms = null;
 	$insert_tops = array(
-		array(1, 'L2jBrasil', 'https://top.l2jbrasil.com', 'l2jbrasil.png', 'l2jbrasil.php', 'sem_id', 'sem_token', 0, 0),
-		array(2, 'Private Server', 'https://www.private-server.ws', 'private_server_ws.jpg', 'privateserverws.php', 'sem_id', 'sem_token', 0, 0),
+		array(1, 'L2jBrasil', 'http://top.l2jbrasil.com', 'l2jbrasil.png', 'l2jbrasil.php', 'sem_id', 'sem_token', 0, 0),
+		array(2, '4TOP MMO', 'https://top.4teambr.com', '4topmmo.png', '4topmmo.php', 'sem_id', 'sem_token', 0, 0),
```

Comment:
You shouldn't remove the existing top list, and add it at the end of the list with 

```php
array(17, '4TOP MMO', 'https://top.4teambr.com', '4topmmo.png', '4topmmo.php', 'sem_id', 'sem_token', 0, 0),
```